.buildkite/pipeline.yml

```diff
@@ -32,6 +32,7 @@ steps:
  - commands:
    - cd docker
    - nix run nixpkgs.docker -c ./docker-static-build.sh
+   - nix run nixpkgs.upx -c upx tezos-*
```

Comment:
I heard you had segfaults with this. Are they gone?

Reply:
it's fine after upx update :yao: